tests/foreman/ui/test_syncplan.py

```diff
@@ -83,14 +39,29 @@ def test_positive_create_basic(self, test_data):
 
         """
         with Session(self.browser) as session:
-            make_syncplan(
-                session,
-                org=self.organization.name,
-                name=test_data['name'],
-                description=test_data['desc'],
-                sync_interval=test_data['interval'],
-            )
-            self.assertIsNotNone(self.syncplan.search(test_data['name']))
+            for value in generate_strings_list():
```

Comment:
@oshtaier ty for the comment.  Now there are three tests - to get name, desc, sync interval separately.  Also code now uses random feature to select the sync interval in name and desc tests.  This should address what you are asking for.